backend/src/project/service.ts

```diff
@@ -31,3 +32,11 @@ export const validateProject = (req: unknown): validProjectOrError => {
         return [false, error];
     }
 };
+
+export const isUserAuthorised = (project: Project, user: string): boolean => {
+    if (project.author === "AlgoDebug" || (user && project.author === user)) {
```

Comment:
No nie jest to błąd, ale nie wygląda to profesjonalnie, a my przecież jesteśmy profesjonalnymi developerami xD.
Zamiast tego `if (warunek) return true; else return false` zrób po prostu `return warunek`.

Reply:
Na marginesie dodam, że ten nawias otaczający `user && project.author === user` nie jest potrzebny. Aczkolwiek jestem zdania, że jak ktoś nie jest pewien, to lepiej już dodać dodatkowy nawias i mieć pewność, że działa tak jak chcieliśmy.